src/pages/EmptyWishlistViewProducts.jsx

```diff
@@ -65,18 +66,165 @@ function EmptyWishlistViewProducts() {
                 <rect width="90.3342" height="90.3507" fill="white" transform="translate(274.283 268.901)"/>
                 </clipPath>
                 </defs>
+
             </svg>
+            <form action="">
+              <input
+                className="bg-gray-50"
+                type="text"
+                placeholder="Search for gadgets"
+              />
+            </form>
+          </div>
+          {/* notificaation icon */}
+          <div className=" bg-gray-300 rounded-full p-3">
+            <svg
+              width="26"
+              height="26"
+              viewBox="0 0 26 26"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M20.9516 15.6975L19.8683 13.8991C19.6408 13.4983 19.435 12.74 19.435 12.2958V9.55496C19.435 7.00912 17.94 4.80996 15.7841 3.78079C15.2208 2.78413 14.1808 2.16663 12.9891 2.16663C11.8083 2.16663 10.7466 2.80579 10.1833 3.81329C8.07081 4.86413 6.60831 7.04163 6.60831 9.55496V12.2958C6.60831 12.74 6.40248 13.4983 6.17498 13.8883L5.08081 15.6975C4.64748 16.4233 4.54998 17.225 4.82081 17.9616C5.08081 18.6875 5.69831 19.2508 6.49998 19.5216C8.60165 20.2366 10.8116 20.5833 13.0216 20.5833C15.2316 20.5833 17.4416 20.2366 19.5433 19.5325C20.3016 19.2833 20.8866 18.7091 21.1683 17.9616C21.45 17.2141 21.3741 16.3908 20.9516 15.6975Z"
+                fill="#717591"
+              />
+              <path
+                d="M16.0658 21.6775C15.6108 22.9341 14.4083 23.8333 13 23.8333C12.1442 23.8333 11.2992 23.4866 10.7033 22.8691C10.3567 22.5441 10.0967 22.1108 9.94501 21.6666C10.0858 21.6883 10.2267 21.6991 10.3783 21.7208C10.6275 21.7533 10.8875 21.7858 11.1475 21.8075C11.765 21.8616 12.3933 21.8941 13.0217 21.8941C13.6392 21.8941 14.2567 21.8616 14.8633 21.8075C15.0908 21.7858 15.3183 21.775 15.535 21.7425C15.7083 21.7208 15.8817 21.6991 16.0658 21.6775Z"
+                fill="#717591"
+              />
+            </svg>
+          </div>
+          {/* User Image */}
+          <img src="../src/assets/icons/avatar.png" alt="user avatar" />
         </div>
-        <div className=" text-center relative bottom-12 md:ml-48 md:relative md:bottom-32">
-            <h1 className="font-cabinetGrotesk text-[20px] font-bold">Your Wishlist is Empty</h1>
-            <p className="font-campton text-[20px] text-gray-500">Add products to your wishlist</p>
-        </div>
-        {/* Create button */}
-        <div className=" flex justify-center pb-10 md:ml-48 md:relative md:bottom-24">
-            <CreateWishButton />
-        </div>
+      </div>
```

Comment:
A code syntax error in this file caused the deployment of the application to fail. Can you check it out?